### ISSUE TITLE
Gets stuck when parsing next time

### ISSUE THREAD

**Issue by mehuldudi** [harrisiirak/cron-parser#134]
The thread gets stuck without error if we try `parser.parseExpression("* * 2 * *", { tz: "America/Sao_Paulo"}).next();`. This happens for this particular timezone with any expression with a defined day.

**Comment by fishcharlie** [comment]
@mehuldudi I can't reproduce this. I'm running the latest version of this package (2.6.0), and running Node.js version (v10.8.0).

Running the following code:

```
const parser = require('cron-parser');
const result = parser.parseExpression("* * 2 * *", { tz: "America/Sao_Paulo"});

console.log(result.next().toDate());
console.log(result.next().toDate());
console.log(result.next().toDate());
```

Prints the following in the console:

```
2018-12-02T02:00:00.000Z
2018-12-02T02:01:00.000Z
2018-12-02T02:02:00.000Z
```

Can you provide more information so that I can reproduce this and I'll try to see if I can help fix it and submit a PR for it?

**Comment by harrisiirak** [comment]
Should be fixed by changes made for #131
